packages/fieldset/test/lion-fieldset.test.js

```diff
@@ -646,6 +646,7 @@ describe('<lion-fieldset>', () => {
       expect(input.modelValue).to.equal('Bar');
 
       el.resetGroup();
+      await nextFrame();
```

Comment:
Would `await el.updateComplete;` suffice? Is a bit cleaner / reliable.

Reply:
decided to use an ` await el.resetGroup();` where possible, seemed even cleaner to me.

Reply:
This definitely makes a lot of sense if you're testing submit followed by reset. 

I see a lot of awaits added to the tests that don't submit prior to the reset, and I'm now wondering why we need to await it in those tests? 

Just because it's now an async method and you can await it, doesn't mean you have to right? Do the tests fail without the await keyword? Because if they do, I would like to understand why they fail, because for most of the tests there should not be a lit update in queue when calling resetGroup.